wait-for-deployment

```diff
@@ -60,4 +76,5 @@ while [[ ${available} -ne ${replicas} ]]; do
   echo "available replicas: ${available}"
 done
 
+kill ${timeout_monitor_pid} #Stop timeout monitor
```

Comment:
This is better implemented as a trap set up immediately after the monitor is executed: After line 58, do

```bash
trap "kill ${timeout_monitor_pid}" EXIT
```

This guarantees that we remove the monitor no matter whether we exit normally, abnormally, or through a bug in the script caught by one of the bash safety options set in the beginning (like `errexit`).